palace/utils/geodata.cpp

```diff
@@ -560,13 +560,12 @@ bool EigenLE(const Eigen::Vector3d &x, const Eigen::Vector3d &y)
 };
 
 // Helper for collecting a point cloud from a mesh, used in calculating bounding boxes or
-// bounding balls.
-std::vector<Eigen::Vector3d> CollectPointCloud(mfem::ParMesh &mesh,
-                                               const mfem::Array<int> &marker, bool bdr)
+// bounding balls. Returns the vector of vertices on the dominant rank, and none on all
+// ranks. Vertices are de-duplicated to a certain floating point precision.
```

Comment:
```
// Helper for collecting a point cloud from a mesh, used in calculating bounding boxes and
// bounding balls. Returns the dominant rank, for which the vertices argument will be
// filled, while all other ranks will have an empty vector. Vertices are de-duplicated to a
// certain floating point precision.
```

Reply:
Fixed in [fa13b91](https://github.com/awslabs/palace/pull/75/commits/fa13b9127936f410ec4f4ddb599c29abac62e848).